src/pages/login/Login.tsx

```diff
@@ -50,27 +50,89 @@ export function Login() {
-          id="login"
-          type="text"
-          className="text"
-          onChange={(e) => handleUpdateUsername(e)}
-        />
+    <div className="container h-screen justify-center max-w-full mx-auto py-24 px-6 bg-blue-100">
+      <div className="font-sans">
+        <div className="max-w-sm mx-auto px-6">
+          <div className="relative flex flex-wrap">
+            <div className="w-full relative">
+              <div className="mt-6">
+                <div className="mb-5 pb-1border-b-2 text-center font-base text-gray-700">
+                  <span>
+                    {" "}
+                    By <span className="text-blue-500">@anythos</span>
+                  </span>
+                </div>
+                <div className="text-center text-2xl sm:text-3xl font-bold">
+                  Sign in
+                </div>
+                <form className="mt-8">
+                  <div className="mx-auto max-w-lg">
+                    <div className="py-2">
+                      <span className="px-1 text-base text-dark-700">
+                        {" "}
+                        Corporate Login{" "}
+                      </span>
+                      <input
+                        id="login"
+                        type="text"
+                        placeholder=""
+                        className="text-md block px-3 py-2 rounded-lg w-full bg-white border-2 border-gray-300 placeholder-gray-600 shadow-md
+                                                focus:placeholder-gray-500 focus:bg-white focus:border-gray-600 focus:outline-none"
+                        onChange={(e) => handleUpdateUsername(e)}
+                      />
+                    </div>
+                    <div className="py-2">
+                      <span className="px-1 text-base text-dark-700">
+                        Password
+                      </span>
+                      <div className="relative">
+                        <input
+                          id="password"
+                          type="password"
+                          placeholder=""
+                          className="text-md block px-3 py-2 rounded-lg w-full bg-white border-2 border-gray-300 placeholder-gray-600 shadow-md
+                                                    focus:placeholder-gray-500 focus:bg-white focus:border-gray-600 focus:outline-none"
+                          onChange={(e) => handleUpdatePassword(e)}
+                        />
+                      </div>
+                    </div>
+                    <div>
+                      <label className="block text-gray-500 my-4 text-right">
+                        <a
+                          href="#"
+                          className=" cursor-pointer tracking-tighter text-gray border-b-2 border-gray-200 hover:border-gray-400"
+                        >
+                          Reset Password
+                        </a>
+                      </label>
+                    </div>
```

Comment:
Please use `label` tag for input instead `span`.

Reply:
changes done